dpe/fuzz/fuzz_targets/runtime_dpe_dpe_fuzz_harness.rs

```diff
@@ -0,0 +1 @@
+../../runtime_dpe_dpe_fuzz_harness.rs
```

Comment:
Is this symlinked? It might make more sense to have a helper crate that both sets of fuzz tests can depend on.

Reply:
Yes. It may be possible to merge all these files into one crate. Otherwise, I suppose so, I could implement that.

Reply:
Marking as resolved, AFL and libFuzzer are being merged, per below.